src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -318,17 +318,17 @@ Some features may not work properly in the old player layout."</string>
     <string name="revanced_ryd_video_likes_hidden_by_video_owner">Hidden</string>
 
 
-    <!-- PreferenceScreen: Return YouTube Username -->
-    <string name="revanced_preference_screen_return_youtube_username_title">Return YouTube Username</string>
+    <!-- PreferenceScreen: Return YouTube username -->
```

Comment:
Idk if I was meant to change this

Reply:
With lowercase U it makes zero sense. How @inotia00 did it was the right way, with uppercase U - as Username.